### ISSUE TITLE
why calculate GWAS effective sample size for binary traits as (4 / (1 / n_case + 1 / n_control)) ?

### ISSUE THREAD

**Issue by chunhuichen** [privefl/bigsnpr#398]
Dear Dr. Prive

Thanks for your great tool LDpred2!

I am learning to use it to calculate PRS, but encountered some questions. Can you help me out ? Thanks!

1, on your webpage https://privefl.github.io/bigsnpr-extdoc/polygenic-scores-pgs.html,  it reads
# GWAS effective sample size for binary traits (4 / (1 / n_case + 1 / n_control))
However, when I checked the PGC ADHD gwas suammary statistics  ( https://figshare.com/articles/dataset/adhd2019/14671965 ), it had 3 columns titled Nca, Nco,Neff, I guess they are sample size of case/control/effective, but the numbers do not follow your equation. for example, 19099   34194   22842.43 in the file, but (4 / (1 / 19099 + 1 / 34194))=49017, which was much larger than Neff 22842.43
The same inconsistent can be found with PGC SCZ summary statistics https://figshare.com/articles/dataset/scz2022/19426775 (53386   77258   58749.13 for case/control/effective)
So I am guess you and PGC used different equation, may I ask why ? and how should I use these numbers?

2, In your paper  LDpred2: better, faster, stronger, you compared LDpred2 with other similar tools and found LDpred2 outperformed others (figue 4). However, I think the AUC did not differ much between these tools. I guess PRS calcualted with all tools would be highly correlated. So do you think all tools are reliable or only LDpred2 should be used?

Thanks for any help!

**Comment by privefl** [comment]
1. Two explanations I guess:
 - The PGC previously used 2 / (1 / Nca + 1 / Nco) as the Neff; in recent sumstats, I think they now call that HalfNeff.
 - In the case of a meta-analysis of many sumstats, Neff_meta should be the sum of Neff for each study, not the Neff computed from the sum of Nca and the sum of Nco (cf. https://doi.org/10.1016/j.biopsych.2022.05.029), which the PGC does right.
 
    TLDR: You should use the column they provide and multiply it by 2 for LDpred2.
    
2. No, LDpred2 is not the only PGS method you can use of course. It is currently one of the best though.

**Comment by chunhuichen** [comment]
Clear.  Thanks!

So I guess when input Neff into other tools (i.e. LDSC), it should also be 4 / (1 / Nca + 1 / Nco) ,right?

**Comment by privefl** [comment]
Yes, but prefer using the sum of Neff instead when available.
